src/main/java/com/d2fn/guano/DumpJob.java

```diff
@@ -92,6 +96,18 @@ private void writeZnode(ZooKeeper zk, String outFile, String znode) throws Excep
                 out.flush();
                 out.close();
             }
+        } else {
+            if (!outFile.contains("_znode") && stat.getEphemeralOwner() == 0) {
+                // Create an empty folder for Ephemeral nodes.
```

Comment:
This is not actually dumping ephemeral nodes. == 0 means it is not ephemeral. Did you intend to dump ephemeral nodes? If so that would need to be optional as in most cases it would be an error to restore ephemeral nodes. If not just correct the comment.